src/applications/check-in/day-of/pages/Error.jsx

```diff
@@ -1,20 +1,27 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 
 import BackToHome from '../../components/BackToHome';
 import Footer from '../../components/layout/Footer';
 
+import { makeSelectError } from '../../selectors';
 import { useSessionStorage } from '../../hooks/useSessionStorage';
 import Wrapper from '../../components/layout/Wrapper';
 
 const Error = () => {
   const { t } = useTranslation();
   const { getValidateAttempts } = useSessionStorage(false);
+  const selectError = useMemo(makeSelectError, []);
+  const { error } = useSelector(selectError);
   const { isMaxValidateAttempts } = getValidateAttempts(window);
+
+  const validationError = isMaxValidateAttempts || error === 'max-validation';
```

Comment:
I love this, and I think we should heavily consider using it for our refector. Could you document this approach a bit and / or link this PR to the refactor Error page ticket for when that gets picked up?

Reply:
Yep, I'll add this to that other ticket!